### ISSUE TITLE
Does it provide level select?

### ISSUE THREAD

**Issue by illusiwind** [argonlefou/KSHG_ResChanger#1]
I noticed that "Level Select" was mentioned in the description, but I didn't find it. The banner is removed, and the leaderboard seems to be working, but how do I select level?
Also, if level can be selected, I wonder if UFO ending can be unlocked? Apparently skipping stages is supposed to be unlocked by e-Amusement Pass, which should be the same for UFO ending. I found the video of UFO ending in the files, but I wonder if there's a way to actually shoot the UFOs in game and get that ending. If level select is possible, is UFO ending also possible?

**Comment by argonlefou** [comment]
The tool itself does not provide any level select feature.

The "Level Select" tool I mentionned in the description is a credit to some tool (and it was not my work) I used back then I was coding this, allowing me to get quickly to a specific late-game level to fix some graphical glitch.

Well, more than a "tool", it was a set of pre-modified `KSHG_no_cursor.exe`, each one starting at a different level
[1 - SilentHill The Arcade Stage Select.zip](https://github.com/argonlefou/KSHG_ResChanger/files/15075330/1.-.SilentHill.The.Arcade.Stage.Select.zip)

:warning: **_Important note : :warning: _** I'm not the author of these files, and I do not know what are inside these exe, so use them at your own risk...


As for the UFO part, I don't think the stage select I pasted above would be able to get it.
If it's really tied to the E-Amusement pass to get loaded data (like already finished game) in order to make UFO appear to shoot them...
